deps/npm/CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+### v2.2.0 (2015-01-08):
```

Comment:
oh look .. here's a changelog for us, I retract that comment about needing a changelog then